gradio/components.py

```diff
@@ -318,7 +336,8 @@ def __init__(
         self.lines = lines
         self.max_lines = max_lines
         self.placeholder = placeholder
-        self.value = self.postprocess(value)
+        load_fn, initial_value = self.get_load_fn_and_initial_value(value)
```

Comment:
You'll see this pattern for getting the load_fn repeated in every IOComponent because the logic for setting the value is done in each component as opposed to the IOComponent parent class.